site/includes/sign-on-off.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+	<div class="row">
+{{#is signed "off"}}
```

Comment:
The register/login interface is the content as it would be a login page, e.g., https://clegc-gckey.gc.ca/j/eng/l?ReqID=s25ee43853adc59e8b95f29aef5afdf398227277ea.